great_expectations/cli/docs.py

```diff
@@ -130,32 +130,12 @@ def build_docs(context, site_name=None, view=True):
     msg = "\nThe following Data Docs sites will be built:\n\n"
     for site_name, index_page_locator_info in index_page_locator_infos.items():
         if index_page_locator_info.startswith("file"):
```

Comment:
This check no longer seems to be doing anything, correct? What is the intended logic of flow here?

Reply:
@jcampbell You are right! This if/else was a leftover - fixed.